### ISSUE TITLE
crash when requesting playlist info

### ISSUE THREAD

**Issue by Vrihub** [mps-youtube/mps-youtube#468]
After searching for a playlist with `//`, if I request information for a playlist using the `i...` command,  mpsyt 
crashes with the following error messages:

    Traceback (most recent call last):
      File "/usr/local/bin/mpsyt", line 9, in <module>
        load_entry_point('mps-youtube==0.2.6', 'console_scripts', 'mpsyt')()
      File "/usr/local/lib/python3.4/dist-packages/mps_youtube/main.py", line 3695, in main
        if matchfunction(i.function, i.regex, userinput):
      File "/usr/local/lib/python3.4/dist-packages/mps_youtube/main.py", line 3644, in matchfunction
        func(*matches)
      File "/usr/local/lib/python3.4/dist-packages/mps_youtube/main.py", line 3072, in info
        ytpl = g.pafy_pls.get(p['link'])[0]
    TypeError: 'NoneType' object is not subscriptable

IIRC this had already happened sporadically in the past, but since today it's happening with ALL playlists. Maybe youtube changed something?

I'm on linux, using mpsyt version 0.2.6, pafy 0.5.0, youtube-dl 2016.04.01 (all installed via pip3 and upgraded); python version 3.4.2

**Comment by ids1024** [comment]
I think commit f95b5523cf596f8a12020443166ca4270a84be16 in the develop branch should fix this.

**Comment by Vrihub** [comment]
I applied your patch and it seems to fix the problem, thank you very much!

BTW sorry for being too alarmist in my report, I should have said the bug was happening with _some_ playlists, not _all_; indeed it was just a gross overgeneralisation, since I only tried 5 playlists and all failed, but later I found some playlists working as expected.
I'll keep on testing and will report if I find other problems.
